themes/powerline-multiline-blue/powerline-multiline-blue.base.bash

```diff
@@ -0,0 +1,144 @@
+# Define this here so it can be used by all of the Powerline themes
+
+. "$BASH_IT/themes/powerline-multiline/powerline-multiline.theme.bash"
+. "$BASH_IT/themes/powerline-multiline/powerline-multiline.base.bash"
```

Comment:
Looks like the [`powerline-multiline.theme.bash`](https://github.com/Bash-it/bash-it/blob/master/themes/powerline-multiline/powerline-multiline.theme.bash) already imports the `powerline-multiline.base.bash` - Wondering if you're double-importing it here?